src/services/findAllReferences.ts

```diff
@@ -526,7 +526,7 @@ namespace ts.FindAllReferences {
     function getTextSpan(node: Node, sourceFile: SourceFile, endNode?: Node): TextSpan {
         let start = node.getStart(sourceFile);
         let end = (endNode || node).getEnd();
-        if (isStringLiteralLike(node)) {
+        if (isStringLiteralLike(node) && (end - start) > 2) {

```

Comment:
This line alone is _technically_ the minimal fix to stop the crash, but it doesn't really get at the root cause (forcibly parsing a non-string-thing as a string) and instead treats the symptom (stopping adjusting the span bounds for strings shorter than a minimum quoted string length).  I'm of a mind to keep this, since I'm pretty sure `getTextSpan` crashes on empty string literals without it, too (because `createTextSpan` demands a nonzero nonnegative span length), though I don't know of a repro for that case.